knowledge-base/grid-kb-style-filtered-columns.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Change Filtered Columns Color in Grid for Blazor
+description: Learn how to change the background color of filtered columns in the Grid for Blazor for better visibility using CSS.
+type: how-to
+page_title: How to Style Filtered Columns in Telerik Grid for Blazor
+slug: grid-kb-style-filtered-columns
+tags: grid, blazor, styling, filtered columns, css, oncellrender, onstatechanged
+res_type: kb
+ticketid: 1672604
+---
+
+## Environment
+
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Grid for Blazor</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to style filtered columns in the Grid for Blazor using RGBA for better visibility. Specifically, when a filter is applied, I want the background color of column cells to change.
+
+This knowledge base article answers the following questions:
+- How to apply custom styles to filtered columns in a Blazor Grid?
+- How to enhance the visibility of filtered columns in Blazor Grid using CSS?
+
+## Solution
+
+To style filtered columns in a Telerik Grid for Blazor: 
+1. Utilize the [`OnCellRender`]({%slug grid-column-events%}#oncellrender) and [`OnStateChanged`]({%slug grid-state%}#onstatechanged) events 
+2. Apply a custom CSS class to the filtered column when a filter is active
+
+>caption Grid with styled filtered column.
+
+`````CSHTML
+@using Telerik.DataSource.Extensions
+@using Telerik.DataSource
+
+<style>
+    .highlighted-column {
+        background-color: rgba(255, 0, 0, 0.2); /* Change to your desired RGBA */
+        color: #191970;
+    }
+</style>
+
+<TelerikGrid TItem="@Employee" 
+             OnRead="@ReadItems"
```

Comment:
Do we need OnRead for the purpose of this example? If not, the code can be simplified and shortened.